examples/guestbook-go/redis-master-service.json

```diff
@@ -12,7 +12,8 @@
       "ports": [
         {
           "port":6379,
-          "targetPort":"redis-server"
+          "targetPort":"redis-server",
+          "nodePort":0
```

Comment:
The failed e2e test ran before this change... now the failure is gone, and I've removed the "nodePort" field from this json file.